### ISSUE TITLE
IconImageSource

### ISSUE THREAD

**Issue by devastaza** [radimitrov/CSharpShellApp#293]
For some reason, the iconimagesource property stopped working, although it worked a couple of updates ago. Source can be specified through the backend (from resources or using the UI.GetAsset() method, but then the icons sometimes change their places or dublicates (there are no problems in the compiler, only in the assembled apk).

There is a bug
![IMG_20230609_005427.jpg](https://github.com/radimitrov/CSharpShellApp/assets/72895389/ec221dd8-6e00-4818-8034-63eb662fd8f0)

But there is a correct version
![IMG_20230609_005446.jpg](https://github.com/radimitrov/CSharpShellApp/assets/72895389/0e8fd132-2b9b-43cb-9358-c1fb7389a9ee)

**Comment by devastaza** [comment]
No, rotating didn't change anything. Icons replace each other or dublicate after reopen the program. If you want, i can send compiled apk or source code.

**Comment by radimitrov** [comment]
@devastaza Probably not a rendering issue then. If possible source code would be essential since the root cause could be anything. Also on what Android version are you?

**Comment by devastaza** [comment]
Sure, i'll send source code soon.

**Comment by radimitrov** [comment]
@devastaza Might not be needed, hopefully. There should be an update by tomorrow which includes changes in the XAML compiler. That might fix it.

**Comment by devastaza** [comment]
So, if fix doesn't help us (hopefully does) i'll send

**Comment by tyronemagwade** [comment]
I've also noticed that if you Binding the image source using the mvvm pattern if doesn't work
